lib/matplotlib/image.py

```diff
@@ -144,7 +144,11 @@ def flush_images():
                     gc = renderer.new_gc()
                     gc.set_clip_rectangle(parent.bbox)
                     gc.set_clip_path(parent.get_clip_path())
-                    renderer.draw_image(gc, round(l), round(b), data)
+                    if type(renderer) == mpl.backends.backend_agg.RendererAgg:
```

Comment:
why?

Reply:
Hadn't thought of third-party backends. With your suggestion, it would be something like...

```
if hasattr(renderer, 'draw_image_2'):
```

but this would effectively be deprecating a pretty core rendering function....so probably not a good idea?